### ISSUE TITLE
KeyError: 'world' when using plot_round()

### ISSUE THREAD

**Issue by danieladugyan** [pnxenopoulos/awpy#169]
**Describe the bug**
Trying to visualize rounds by using `plot_round()`, but getting stuck on this error. As far as I can tell, there isn't supposed to be a "world" key, so I don't know why there's a reference to it in the [source code](https://github.com/pnxenopoulos/awpy/blob/d862e6b17251f8be8a349b81a49384bf20fe1d35/awpy/visualization/plot.py#L132).

**Include Information to Reproduce**
OS: Windows
Follow the steps in [example 02](https://github.com/pnxenopoulos/awpy/blob/main/examples/02_Basic_CSGO_Visualization.ipynb).
```python
# main.py
from awpy import DemoParser
from awpy.visualization.plot import plot_round

demo_parser = DemoParser(demofile="2836585.dem", demo_id="2836585", parse_rate=128)

data = demo_parser.parse()

plot_round("output.gif", data["gameRounds"][5]["frames"], map_name=data["mapName"], map_type="simpleradar")
```
```bash
> python main.py
0it [00:00, ?it/s]
Traceback (most recent call last):
  File "main.py", line 8, in <module>
    plot_round("output.gif", data["gameRounds"][5]["frames"], map_name=data["mapName"], map_type="simpleradar")
  File ".env\lib\site-packages\awpy\visualization\plot.py", line 132, in plot_round
    for object_type in f["world"]:
KeyError: 'world'
```

**Comment by pnxenopoulos** [comment]
Has to do with recent change away from `"world"` key in frames. Now "bomb" has its own x/y/z.

Will have to fix this function and the associated notebook.

**Comment by pnxenopoulos** [comment]
Fixed in #173
